packages/legend-graph/src/graph-manager/action/changeDetection/DomainObserverHelper.ts

```diff
@@ -526,6 +526,30 @@ export const observe_FunctionTestSuite = skipObservedWithContext(
     return metamodel;
   },
 );
+
+export const observe_ConcreteFunctionDefinitionWithoutTests = skipObserved(
```

Comment:
what is the reason for needing this separate observe function without tests ?

Reply:
The reason I added another one, is because the place I call this in the mapping editor does not have a context, which is being used in the old function. Since I don't need tests, hence added this func

Reply:
for it the context is a required parameter, which I don't have at that point

Reply:
thanks! will change this